### ISSUE TITLE
Wrong user updated during login

### ISSUE THREAD

**Issue by Swiff** [Adldap2/Adldap2-Laravel#257]
- Laravel Version: 5.4.12 and 5.1.9
- Adldap2-Laravel Version: 3.0.0
- PHP Version: 7.0.15

### Description:
When having a single user in an active directory and using adldap2-laravel to authenticate, the user gets created / updated fine.

If I try another user from the AD, the first user created in the table gets updated.

See below log from MariaDB

> 66 Prepare	select * from `users` limit 1
> 66 Execute	select * from `users` limit 1
> 66 Close stmt
> 66 Prepare	update `users` set `name` = ?, `email` = ?, `username` = ?, `password` = ?, `updated_at` = ? where `id` = ?
> 66 Execute	update `users` set `name` = 'Bo Bjørn', `email` = 'bbjoern@acme.org', `username` = 'bbjoern', `password` = '$2y$10$TQBlVFSrX5t4jjRiYJwq5eAL2CSPrEe4AwjMJb1ZRiZ61z7AAgibq', `updated_at` = '2017-02-16 08:52:47' where `id` = 1

### Steps To Reproduce:
adldap_auth.php:
```php
<?php
return [
    'connection' => env('ADLDAP_CONNECTION', 'default'),
    'provider' => Adldap\Laravel\Auth\DatabaseUserProvider::class,
    'resolver' => Adldap\Laravel\Auth\Resolver::class,
    'importer' => Adldap\Laravel\Auth\Importer::class,
    'rules' => [
        Adldap\Laravel\Validation\Rules\DenyTrashed::class,
    ],
    'scopes' => [
        Adldap\Laravel\Scopes\UpnScope::class,
    ],
    'usernames' => [
        'ldap' => 'samaccountname',
        'eloquent' => 'username',

    ],
    'login_fallback' => env('ADLDAP_LOGIN_FALLBACK', false),
    'password_sync' => env('ADLDAP_PASSWORD_SYNC', true),
    'windows_auth_attribute' => ['samaccountname' => 'AUTH_USER'],
    'sync_attributes' => [
        'email' => 'mail',
        'name' => 'cn',
        'username' => 'samaccountname'
    ],
];
```

**Comment by moth** [comment]
I think this is the same issue as mine on #258

**Comment by stevebauman** [comment]
Fixed in `v3.0.1`, please run `composer update`.

Thanks @moth and @Swiff for making me aware of the issue!

**Comment by Swiff** [comment]
Great stuff, working now.

Thank you for a quick fix!
